src/main/java/com/hbm/dim/SolarSystem.java

```diff
@@ -132,8 +133,102 @@ public static void init() {
 							.withSemiMajorAxis(179_890)
 							.withRotationalPeriod(901_902)
 
-					)
+					),
+					new CelestialBody("sarnus")
+						.withMassRadius(1.223e24F, 5_300)
+						.withSemiMajorAxis(125_798_522)
+						.withRotationalPeriod(28_500)
+						.withColor(1f, 0.6862f, 0.5882f)
+						.withSatellites(
+
+						new CelestialBody("hale")
+							.withMassRadius(1.2166e16F, 6)
+							.withSemiMajorAxis(10_488)
+							.withRotationalPeriod(23_555),
+
+						new CelestialBody("ovok")
+							.withMassRadius(4.233e17F, 26)
+							.withSemiMajorAxis(12_169)
+							.withRotationalPeriod(29_440),
+
+						new CelestialBody("eeloo")
+							.withMassRadius(1.115e21F, 210)
+							.withSemiMajorAxis(19_106)
+							.withRotationalPeriod(57_915),
+
+						new CelestialBody("slate")
+							.withMassRadius(2.965e22F, 540)
+							.withSemiMajorAxis(42_593)
+							.withRotationalPeriod(192_771),
+							
+						new CelestialBody("tekto")
+							.withMassRadius(2.883e21F, 280)
+							.withSemiMajorAxis(97_355)
+							.withRotationalPeriod(666_154)
+							.withTraits(new CBT_Atmosphere(Fluids.TEKTOAIR, 1.2463F))
+
+					),
+				new CelestialBody("urlum")
```

Comment:
Very cool stuff, can we add Uranus' axial tilt to Urlum? (It'll show up in the sky rotated even though we can't see what it's like from the surface)